keen/keen-dataviz.js
# Display a message in a table row if there is no data to display

## What does this PR do? How does it affect users?

This PR displays a message when empty tables are rendered in keen-dataviz. This way a user doesn't need to look at a blank slate every time they load a table with no data.

## How should this be tested?

Look at the "Empty Table" example in `test/demo/index.html`. Make sure a message is displayed for an empty dataset.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?

https://metakeen.atlassian.net/secure/RapidBoard.jspa?rapidView=28&view=detail&selectedIssue=EXP-469


## Screenshots

### Before
![screen shot 2016-10-31 at 1 46 19 pm](https://cloud.githubusercontent.com/assets/638351/19869097/7a990314-9f70-11e6-8c5b-94d560ab6568.png)

### After
![screen shot 2016-12-20 at 1 45 25 pm](https://cloud.githubusercontent.com/assets/638351/21367752/90226914-c6bc-11e6-8420-964875e3827f.png)
